certified-connectors/Spinpanel/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2047 @@
+                        "format": "int32",
+                        "description": "Consumed units for this sku.",
+                        "title": "Consumed units"
+                      },
+                      "id": {
+                        "type": "string",
+                        "description": "Id of the subscribed sku object.",
+                        "title": "Id"
+                      },
+                      "skuId": {
+                        "type": "string",
+                        "description": "Sku id of the Microsoft license.",
+                        "title": "Sku id"
+                      },
+                      "skuPartNumber": {
+                        "type": "string",
+                        "description": "Sku partnumber of the Microsoft license.",
+                        "title": "Sku partnumber"
+                      },
+                      "appliesTo": {
+                        "type": "string",
+                        "description": "Applies to user or organization.",
+                        "title": "Applies to"
+                      },
+                      "prepaidUnits": {
+                        "type": "object",
+                        "properties": {
+                          "enabled": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Enabled subscribed skus.",
+                            "title": "Enabled"
+                          },
+                          "suspended": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Suspended subscribed skus.",
+                            "title": "Suspended"
+                          },
+                          "warning": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Subscribed skus with warning.",
+                            "title": "Warning"
+                          }
+                        },
+                        "description": "prepaidUnits"
+                      },
+                      "servicePlans": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "servicePlanId": {
+                              "type": "string",
+                              "description": "servicePlanId"
+                            },
+                            "servicePlanName": {
+                              "type": "string",
+                              "description": "servicePlanName"
+                            },
+                            "provisioningStatus": {
+                              "type": "string",
+                              "description": "provisioningStatus"
```

Comment:
@VincentMejan All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
Updated descriptions